diff --git a/src_3rd/sphash/sphash.c b/src_3rd/sphash/sphash.c
index bb9d552..224b96b 100644
--- a/src_3rd/sphash/sphash.c
+++ b/src_3rd/sphash/sphash.c
@@ -300,7 +300,7 @@ void dump_file(char *file_name_, char *prefix)
 
 	f = fopen(file_name, "w");
 	assert(f != NULL);
-	fprintf(f, banner);
+	fprintf(f, "%s", banner);
 	fprintf(f, "#include <stdlib.h>\n");
 	fprintf(f, "#include <string.h>\n");
 	if (aux_fields != NULL) {
@@ -407,7 +407,7 @@ void dump_file(char *file_name_, char *prefix)
 
 	file_name[l+1] = 'h';
 	f = fopen(file_name, "w");
-	fprintf(f, banner);
+	fprintf(f, "%s", banner);
 	file_name__ = sphash_strdup(file_name_);
 	for(s = file_name__; *s != '\0'; s++) {
 		if (((*s >= 'a') && (*s <= 'z')) || ((*s >= 'A') && (*s <= 'Z')) || ((*s >= '0') && (*s <= '9')))
