diff --git a/share/scheme/breeze2ps.scm b/share/scheme/breeze2ps.scm
index 7bca0b4..abb4083 100644
--- a/share/scheme/breeze2ps.scm
+++ b/share/scheme/breeze2ps.scm
@@ -704,11 +704,11 @@
 		 ; prune-file/run-dot : helper procedures for later stuff
 		 (prune-file (lambda (filename)
 			(if (not breeze2ps-keep-dot-files)
-				(system (string-append "/bin/rm -f " filename))
+				(system (string-append "@coreutils@/bin/rm -f " filename))
 			)
 		 ))
 		 (run-dot (lambda (dot-file ps-file)
-			(system (string-append "dot -Tps " dot-file " > " ps-file))
+			(system (string-append "@graphviz@/bin/dot -Tps " dot-file " > " ps-file))
 		 ))
 		)
 		(if breeze2ps-single-file
